reactor-core/src/main/java/reactor/core/publisher/MonoUsing.java

```diff
@@ -116,7 +116,8 @@ public void subscribe(CoreSubscriber<? super T> actual) {
 
 	@Override
 	public Object scanUnsafe(Attr key) {
-		return null; //no particular key to be represented, still useful in hooks
+		if (key == Attr.RUN_STYLE) return Attr.RunStyle.SYNC;
```

Comment:
should also cover `MonoUsingSubscriber`

Reply:
Fixed in commit [0542a81](https://github.com/reactor/reactor-core/pull/2123/commits/0542a81a2f3a7548cfd69739a77d273dd98c4b33)